### DISCUSSION TITLE
Possible to add a notification/confirmation on save?

### DISCUSSION THREAD

**Discussion by JorisOrangeStudio** [statamic/cms#9853] [Q&A]
What is the best practice for adding a confirmation modal upon saving an entry? This would inform the user to also update the localized variants of the page. After changing something on the default language this modal should show up to make sure the other sites (languages) are also updated.

**Comment by duncanmcclean** [comment]
You could maybe use the [`entry.saved` JavaScript hook](https://statamic.dev/extending/js-hooks#entrysaved)?

**Comment by dennis6101990leon** [comment]
Hello,

Absolutely, Best practice is to use a confirmation modal after saving edits to the [HealthCareGov](https://www.health-insurancemarket.com) default language entry. This modal should remind users to update localized variants to ensure consistency across languages.
